R/roxygen-examples-add-remove.R

```diff
@@ -30,15 +30,29 @@ remove_roxygen_mask <- function(text) {
 #' @keywords internal
 remove_roxygen_header <- function(text) {
   text <- gsub("^\\s*@examples\\s*", "", text, perl = TRUE)
-  starts_with_blank <- text[1] == "\n"
-  c(text[1][!starts_with_blank], text[-1])
+  if (grepl("^If ", text[1])) {
```

Comment:
This matches `@examples If` . Just sayin...

Can you use the regex from https://github.com/r-lib/styler/pull/743/files#diff-73645d1e1e747d5eeb45cbd47ddb2ef64c658ed5c585763efd004a1b08ba61f9R10, perhaps with rematch2 and a grouped match?

Reply:
Good catch. I didn't ever figure out how to use `{rematch2}` (and did not bother spending time), maybe I should... For now, I think using the regex you linked should be sufficient.